src/test_misp.py

```diff
@@ -1,6 +1,6 @@
 import argparse
-from cgi import log
-from pymisp import MISPEvent, PyMISP, MISPObject
+from distutils.log import log
```

Comment:
Missed that earlier: should be:

```
import logging

...

logging.info("...")
```

(https://docs.python.org/3/library/logging.html) (possibly with `logging.basicConfig(level=logging.INFO)`)